src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_structure.py

```diff
@@ -39,6 +38,12 @@
 )
 
 
+def random_uuid_str() -> str:
+    from allotropy.parsers.moldev_softmax_pro.softmax_pro_parser import SoftmaxproParser
+
+    return SoftmaxproParser.random_uuid_str()
```

Comment:
@bdworth should this function actually depend on `SoftpaxproParser` implementation of `random_uuid_str`?

What if the parser is deprecated in the future? I personally think parsers should be independent from each other.

Reply:
also applies for `src/allotropy/parsers/perkin_elmer_envision/perkin_elmer_envision_structure.py`